pydantic_ai_slim/pyproject.toml

```diff
@@ -62,7 +62,7 @@ vertexai = ["google-auth>=2.36.0", "requests>=2.32.3"]
 anthropic = ["anthropic>=0.49.0"]
 groq = ["groq>=0.15.0"]
 mistral = ["mistralai>=1.2.5"]
-bedrock = ["boto3>=1.34.116"]
+bedrock = ["boto3>=1.35.74"]
```

Comment:
I had to change the minimum boto3 version because `video` was introduced in version `1.35.74`.

Reference: https://github.com/boto/boto3/blob/develop/CHANGELOG.rst

Reply:
That's fine. 🙏